src/components/pages/Home/RenderHomePage.js

```diff
@@ -80,6 +80,7 @@ function RenderHomePage(props) {
   const [selectedRows, setSelectedRows] = useState({});
   const [hrfUserInfo, setHrfUserInfo] = useState([]);
   const [myPendingCases, setMyPendingCases] = useState([]);
+  const [hasUpdated, setHasUpdated] = useState(false);
```

Comment:
Nice usage of React's `useState` hook to manage update detection - this could theoretically be used elsewhere in the app if necessary and utilized carefully - good work here!